com.unity.sg2/Editor/GraphUI/GraphElements/ModelUI/Inspector/ListPropertyField.cs

```diff
@@ -116,24 +120,24 @@ void OnItemsAdded(IEnumerable<int> obj)
             // Create the dropdown menu, add items and show it
             GenericMenu menu = new GenericMenu();
```

Comment:
I'd imagine that it wouldn't be too hard to lock all this context menu behavior stuff behind a flag and just to use the default behavior that the list view uses, which is that when you add an item, a default entry just gets added to the list, and this sounds more or less exactly like what I see the subgraph output stuff needing, lets chat tomorrow real quick and get aligned on this